app/scripts/services/builds.js

```diff
@@ -145,6 +146,61 @@ angular.module("openshiftConsole")
       return moment(endTimestamp).diff(moment(startTimestamp));
     };
 
+    // return only builds that are incomplete
```

Comment:
Mm... lemme eliminate the comments that tell you exactly the same thing as the function names :confused: